Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2221,6 +2381,25 @@ bool IsSubfolder(string parentPath, string childPath)
         return false;
     }
 
+    public virtual void ClearInstancePrefabs() {
```

Comment:
BaseMeshSync is a public class, so this becomes a public API.
Is this expected ?
And do we need to make it virtual ?

"private protected" is an alternative.